### DISCUSSION TITLE
Encryption and Cryptogrpahy

### DISCUSSION THREAD

**Discussion by Lima9092** [ukncsc/zero-trust-architecture#61] [Ideas]
I think a section on the value of encryption in a zero-trust network and recommendations on securing cryptography would be valuable here

**Comment by grahamietf** [comment]
Hi

I think that the latest version covers the crypto operations quite well. My only nit is that the doc talks about identities and doesn't mention about these being cryptographically verifiable. I've opened pull requests 70,71 and 72 to address this.
